Sourcery/CodeGenerated/Diffable.generated.swift

```diff
@@ -1,127 +1,150 @@
-// Generated using Sourcery 0.4.7 — https://github.com/krzysztofzablocki/Sourcery
+// Generated using Sourcery 0.4.8 — https://github.com/krzysztofzablocki/Sourcery
 // DO NOT EDIT
 
+
 extension Enum {
-     override func diffAgainst(_ object: Any?) -> DiffableResult {
+    override func diffAgainst(_ object: Any?) -> DiffableResult {
         let results = DiffableResult()
-
         guard let rhs = object as? Enum else {
-          results.append("Incorrect type <expected: Enum, received: \(type(of: object))>")
-          return results
+            results.append("Incorrect type <expected: Enum, received: \(type(of: object))>")
```

Comment:
Should I do any additional setup for it? After installing it it started to complain about all lowercased cases